float64 vel_sp
float64 vel_ref
float64 vel_meas